react/ProductList.tsx

```diff
@@ -111,8 +112,37 @@ const ProductGroup: StorefrontFunctionComponent<Props> = props => {
   )
 }
 
+const countModeHandle = (countMode: TotalItemsType, arr: Item[]) => {
```

Comment:
Update this function's name to match the change made in https://github.com/vtex-apps/minicart/pull/249

Reply:
@victorhmp its done! ;)